### DISCUSSION TITLE
How do you plan to work against manifest v3?

### DISCUSSION THREAD

**Discussion by Raj2032** [Eloston/ungoogled-chromium#2884] [General]
This will hurt a lot of extensions especially ublock origin etc. How does ungoogled-chromium plan to work around this? Does it plan to continue off using a version of chromium before manifest v3?

**Comment by PF4Public** [comment]
It has already been answered in https://github.com/ungoogled-software/ungoogled-chromium/issues/662#issuecomment-1850192208. Nothing has changed since then.

**Reply by Raj2032** [reply]
ok thanks!
